### PR TITLE
docs: add release notes for 0.19.10

### PR DESCRIPTION
**Pull request by hanyucui**
Add release notes for 0.19.10.

### CODE REVIEW

**Review by dzhu — APPROVED** [review]
Also add a new item as a breaking change:

```
- The name of the resource pool in Kubernetes has changed from ``"kubernetes"`` to ``"default"``. Forked experiments will need to have their configurations manually modified to update the resource pool name.
```

File: docs/release-notes.rst [hunk 1]
```diff
@@ -10,6 +10,64 @@
...
+-  Notebooks: default idle notebook termination timeout can now be set via ``notebook_timeout``
+   master config option.
+
+-  Trials: Trials can now be killed when in the ``STOPPING_CANCELED`` state. Previously if a trial
+   did not implement preemption correctly and was canceled the trial did not stop and was unkillable
+   until the preemption timeout of an hour.
+
+**Bug Fixes**
+
+-  Fix a bug where Notebooks, Tensorboards, Shells, Commands being restored from a master restart
```
[the remark this answers is not in the document]
Comment (dannysauer) [reply docs/release-notes.rst:50]: Are restored, are not restored? :)

**Comment by hanyucui** [comment]
@dzhu Any idea about this error:
```
git ls-files -z '*.rst' | xargs -0 rstfmt -w 100 --check
:28: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
```

**Comment by mackrorysd** [comment]
> @dzhu Any idea about this error:
> 
> ```
> git ls-files -z '*.rst' | xargs -0 rstfmt -w 100 --check
> :28: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
> ```

It would prefer a long line to a wrapped string in back-ticks. If you just make everything between the backticks one line and then run "make fmt-docs" it formats it differently and should then pass.

**Comment by hanyucui** [comment]
Screen Shot 2023-01-20 at 11 12 59 AM

**Comment by hanyucui** [comment]
Screen Shot 2023-01-20 at 11 19 06 AM

**Pull request merged**